backend/tests/conftest.py

```diff
@@ -378,6 +381,28 @@ def finalized_snapshot(self):
         rv = self._flask_client.post("/snapshots/finalized").text
         return json.loads(rv)
 
+    def get_total_effective_estimated(self):
```

Comment:
`conftest.py` is already huge and we have a task in the backlog to split it into chunks. Let's not add more to this file - could we move a code related to API test to other module?

Reply:
I'd suggest to create one function which gets an URL as a param to reduce code duplication.
```
def get(self, url: str) -> tuple[str, int]:
        rv = self._flask_client.get(url)
        return json.loads(rv.text), rv.status_code
```
 Also, next step might be to keep all the endpoints URLs in one file in `shared` and reuse it between the test and API definitions